### DISCUSSION TITLE
e-mail notification syntax

### DISCUSSION THREAD

**Discussion by waligoras** [theyosh/TerrariumPI#792] [Q&A] [answered]
Hello,
Kinda unexperienced whit coding - can anyone post some examples for email notification, I'm trying to set up an email nofitication when:
1. humidity sensor drops below alarm (or given value if possible)
2. button get's pressed

much appreciated :)

**Comment by theyosh** [comment, marked as answer]
Hi, sorry for the long delay. But I made some more documentation which hopefully will answer your questions: https://theyosh.github.io/TerrariumPI/notifications/
